src/config.rs

```diff
@@ -160,7 +161,7 @@ pub enum SourceFile {
     /// A file.
     File(PathBuf),
     /// A subgroup.
-    Group(Box<Sources>),
```

Comment:
Why exactly do we remove the `Box`? This is again more an understanding question than something that actually matters. I assume we could have a veeeery slight performance lose because of that. Am I right in that assumption?

My ref: 
https://doc.rust-lang.org/book/ch15-01-box.html

I check it would still work with using `Box`

Reply:
I think it makes sense to keep the box, so I reverted the commits. Thanks for the quick review!